data/json/recipes/recipe_food.json

```diff
@@ -5635,6 +5635,19 @@
       [ [ "water", 6 ], [ "water_clean", 6 ] ]
     ]
   },
+  {
+    "type": "recipe",
+    "result": "sports_drink",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_DRINKS",
+    "book_learn": [ [ "textbook_chemistry", 1 ], [ "manual_first_aid", 1 ] ],
+    "skill_used": "cooking",
+    "time": "3 m",
+    "charges": 8,
+    "autolearn": false,
```

Comment:
No need to specify "autolearn" being false for recipes; it's the default value (that is, if you don't specify "autolearn" being true).